packages/eslint-config-terra/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Added
```

Comment:
I believe this is a lint warning. The sections need a line break between the section and first entry. There's an VS code plugin you can download the double check and keep an eye on these things

Reply:
https://github.com/cerner/terra-toolkit/pull/424/commits/540a20f8e567cd2ce41fec3c48b752efe19253df